snippets/curl/nb_images.sh

```diff
@@ -0,0 +1,79 @@
+!/bin/sh
+
+#####################n#####################################################
+
+# This script demostrates the usage of netbackup REST API for listing 
```

Comment:
Minor: Typo - demonstrates.

Minor: Can include a link to jq
https://github.com/stedolan/jq/releases

Reply:
corrected